ansible_collections/arista/avd/plugins/filter/esi_management.py

```diff
@@ -26,6 +26,8 @@ def generate_route_target(self, esi_short):
         str
             String based on route-target format like 03:03:02:02:01:01
         """
+        if esi_short is None:
+            return None
```

Comment:
It's good to return None if esi_short is None.

But I have a doubt, what if the user doesn't input 3 octets or more than 3 octets. Should we handle that as well or just leaving this to the user?

Reply:
Currently, we haven't handled this condition, but yes we can fix as a bug in other PR.